test/python/compiler/test_assembler.py

```diff
@@ -632,9 +632,9 @@ def test_assemble_parametric(self):
         self.assertDictEqual(qobj_insts[2].parameters, {'duration': 25, 'amp': 1})
         self.assertDictEqual(qobj_insts[3].parameters,
                              {'duration': 150, 'sigma': 8, 'amp': 0.2, 'width': 140})
-        self.assertListEqual(
+        self.assertEqual(
             qobj.to_dict()['experiments'][0]['instructions'][0]['parameters']['amp'],
-            [0.0, 0.5])
+            0.0+0.5j)
```

Comment:
I think a nice part of complex number being complex is that we can delegate the handle of them to python. This should work, right?
```suggestion
            0.5j)
```